ddl/ddl_worker.go

```diff
@@ -98,20 +98,20 @@ func (w *worker) typeStr() string {
 }
 
 func (w *worker) String() string {
-	return fmt.Sprintf("%d, tp %s", w.id, w.typeStr())
+	return fmt.Sprintf("worker %d, tp %s", w.id, w.typeStr())
 }
 
 func (w *worker) close() {
 	close(w.quitCh)
 	w.delRangeManager.clear()
 	w.wg.Wait()
-	log.Infof("[ddl] close DDL worker %s", w)
+	log.Infof("[ddl-%s] close DDL worker", w)
 }
 
 // start is used for async online schema changing, it will try to become the owner firstly,
 // then wait or pull the job queue to handle a schema change job.
 func (w *worker) start(d *ddlCtx) {
-	log.Infof("[ddl] start DDL worker %s", w)
+	log.Infof("[ddl-%s] start DDL worker", w)
```

Comment:
I think keeping the previous format is better. What is the difference?

Reply:
[ddl] -> [ddl-worker 0, tp general]

Reply:
So what is the problem with the previous log?

Reply:
Other places need to add `[ddl-worker 0, tp general]`. I want to unify the log, so do it.